modules/cachedb_local/cachedb_local_replication.c

```diff
@@ -0,0 +1,256 @@
+/*
+ * memory cache system replication
+ *
+ * Copyright (C) 2018 Fabian Gast
+ *
+ * This file is part of opensips, a free SIP server.
+ *
+ * opensips is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version
+ *
+ * opensips is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+ *
+ * History:
+ * --------
+ *  2018-12-06  initial version (Fabian Gast)
```

Comment:
Please drop this kind of history - we are in progress of removing them as the GIT logs are doing a much better job on this :)

Reply:
This is removed now.